content/en/docs/instrumentation/js/resources.md

```diff
@@ -66,11 +66,13 @@ In the above example, the SDK detected the process and also added the
 Below you will find instructions to get resources detected automatically for
 you. However, you might run into the situation that no detector exists for the
 resource you need. In that case you can use the environment
-`OTEL_RESOURCE_ATTRIBUTES` to inject whatever you need. For example the
-following script adds [Service][], [Host][] and [OS][] resource attributes:
+`OTEL_RESOURCE_ATTRIBUTES` to inject whatever you need. You can use the
```

Comment:
```suggestion
`OTEL_RESOURCE_ATTRIBUTES` to inject whatever you need. Additionally you can use the
```

minor suggestion otherwise LGTM!

Reply:
There should be a comma after "Additionally".